nexus/db-model/src/inventory.rs

```diff
@@ -2131,17 +2370,53 @@ impl InvOmicronSledConfig {
         generation: external::Generation,
         remove_mupdate_override: Option<MupdateOverrideUuid>,
         host_phase_2: HostPhase2DesiredSlots,
+        measurements: OmicronMeasurements,
     ) -> Self {
         Self {
             inv_collection_id: inv_collection_id.into(),
             id: id.into(),
             generation: Generation(generation),
             remove_mupdate_override: remove_mupdate_override.map(From::from),
             host_phase_2: host_phase_2.into(),
+            measurements: measurements.into(),
         }
     }
 }
 
+#[derive(Queryable, Clone, Debug, Selectable, Insertable)]
+#[diesel(table_name = inv_omicron_sled_config)]
+pub struct DbOmicronMeasurements {
+    pub measurements: Option<Vec<ArtifactHash>>,
+}
+
+impl From<OmicronMeasurements> for DbOmicronMeasurements {
+    fn from(value: OmicronMeasurements) -> Self {
+        let remap = |desired| match desired {
+            OmicronMeasurementSetDesiredContents::InstallDataset => None,
+            OmicronMeasurementSetDesiredContents::Artifacts { hashes } => {
```

Comment:
Is `hashes` guaranteed to be non-empty? Would we rather have a `None` here or a `Some(vec![])`? I'd probably add a comment either way.